documentation/library_guide/notices_disclaimers.rst

```diff
@@ -15,6 +15,8 @@ The products described may contain design defects or errors known as errata whic
 
 Intel disclaims all express and implied warranties, including without limitation, the implied warranties of merchantability, fitness for a particular purpose, and non-infringement, as well as any warranty arising from course of performance, course of dealing, or usage in trade.
 
+Intel, the Intel logo, and Arc are the trademarks of Intel Corporation or its subsidiaries.
+
```

Comment:
We already have a very similar claim at the line 10 above. Is it not sufficient?

Reply:
Perhaps we can just add "Arc" to the line 10 if we just want to call it out specifically?

Reply:
I double-checked the names database, and I made a mistake. This should be a footnote on the page where Arc products are cited, not on the Notices and Disclaimers page. 

This line can be removed from the Notices and Disclaimers page and moved to a footnote on the Release Notes.

Reply:
Done.

Reply:
LGTM.